### ISSUE TITLE
Should support DATE in EXDATE fields

### ISSUE THREAD

**Issue by brianmay** [fmeringdal/rust-rrule#146]
I am getting this warning:

```
Apr 21 10:14:29 iot2 robotica-backend[323758]: 2026-04-21T00:14:29.543705Z  WARN rrule::parser::content_line::date_content_line: Parameter `DATE` is not supported for property name: `EXDATE`. The dates will be interpreter with the `DATE-TIME` parameter instead.
```

These should be supported.

Here is an example entry:

```ical
BEGIN:VCALENDAR
VERSION:2.0
PRODID:-//Test//EN
X-WR-TIMEZONE:America/New_York
BEGIN:VEVENT
DTSTART;VALUE=DATE:20260401
DTEND;VALUE=DATE:20260402
RRULE:FREQ=DAILY;COUNT=5
EXDATE;VALUE=DATE:20260403
DTSTAMP:20260401T000000Z
SUMMARY:Test Event
UID:test@example.com
END:VEVENT
END:VCALENDAR
```

This file passes the validator at https://icalendar.org/validator.html
